backend/prisma/seed.mjs

```diff
@@ -72,7 +73,54 @@ const demoData = {
         'Saturday',
         'Sunday',
       ],
-      ticketPrice: 20,
+    },
+  ],
+  users: [
```

Comment:
@Vaibhavsahu2810 we already have users, tickets, and orders keys, yet you have added the objects by creating their duplicate keys. Can you explain the reason?